app/script/calling/handler/CallStateHandler.coffee

```diff
@@ -137,7 +137,7 @@ class z.calling.handler.CallStateHandler
       else
         @delete_call call_et.id
     .catch (error) =>
-      if error.type is z.calling.CallError::TYPE.CALL_NOT_FOUND
+      if error.message is z.calling.CallError::TYPE.CALL_NOT_FOUND
```

Comment:
Should we skip the `TYPE` and shorten it to `z.calling.CallError::CALL_NOT_FOUND`?